src/BuoyancyModels/buoyancy.jl

```diff
@@ -28,22 +29,10 @@ model = NonhydrostaticModel(grid=grid, buoyancy=buoyancy, tracers=:b)
 ```
 """
 function Buoyancy(; model, vertical_unit_vector=ZDirection())
-    vertical_unit_vector = validate_vertical_unit_vector(vertical_unit_vector)
+    vertical_unit_vector = validate_unit_vector(vertical_unit_vector)
```

Comment:
makes sense to me to move `validate_unit_vector` elsewhere. Good idea.

Reply:
Thanks :)